modules/keda/values.yaml

```diff
@@ -1,2 +0,0 @@
-nodeSelector:
-  tenantname: ${namespace}
```

Comment:
This was moved to a set rather than a values file with a single entry.  If a user passes in a tenant name, it will ultimately get set using the dynamic sets block.